client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/HotRodOperation.java

```diff
@@ -39,6 +39,7 @@
 @Immutable
 public abstract class HotRodOperation<T> extends CompletableFuture<T> implements HotRodConstants, Runnable {
    private static final Log log = LogFactory.getLog(HotRodOperation.class);
+   private static final boolean trace = log.isTraceEnabled();
```

Comment:
This appears to be unused?

Reply:
Yeah, I added a temporary trace log while testing and forgot to remove the field